assets/scss/modules/_forms.scss

```diff
@@ -337,6 +337,10 @@ input[type="tel"] {
   background-repeat: no-repeat;
   font-family: $dp-font-family-proximanova;
 
+  button, a{
+    text-decoration: underline;
+  }
+
 /* Validations Unexpected message */
 &.dp-error {
```

Comment:
This is not part of this PR but if `&.dp-error` is inside `.form-message` the tabulation may be a bit off. The same with the other classes below.